docker-compose.yml

```diff
@@ -4,6 +4,7 @@ networks:
     driver: bridge
 services:
   feeder:
+    image: feederprxorg_feeder
```

Comment:
Makes sure that the image names match between worker and here. The default image name no longer follows this pattern/format so pinning it here makes sure they all match.